vardbg/output/video_writer/default_config.toml

```diff
@@ -45,3 +45,10 @@ heading_size = 24
 # Intro font, used to render the intro screen
 intro = "$ASSETS/fonts/Inter-Bold.ttf"
 intro_size = 48
+
+
+[theme]
+# Theme Name for Vardbg
+# Theme Name List = ['default', 'emacs', 'friendly', 'colorful', 'autumn', 'murphy', 'manni', 'monokai', 'perldoc', 'pastie', 'borland', 'trac', 'native', 'fruity', 'bw', 'vim', 'vs', 'tango', 'rrt', 'xcode', 'igor', 'paraiso_light', 'paraiso_dark', 'lovelace', 'algol', 'algol_nu', 'arduino', 'rainbow_dash', 'abap', 'solarized_dark', 'solarized_light', 'sas', 'stata', 'stata_light', 'stata_dark']
+
+video_theme = "monokai"
```

Comment:
`video` is redundant because this is already a video-specific config file. A more specific key like `color_scheme` would work better as there could be other aspects that one might want to theme.